radix-engine/src/model/transaction_processor/executables.rs

```diff
@@ -609,31 +609,20 @@ impl TransactionProcessor {
                     })
                     .map(|rtn| IndexedScryptoValue::from_typed(&rtn))
                     .map_err(InvokeError::Downstream),
-                Instruction::Basic(BasicInstruction::PublishPackageWithOwner {
```

Comment:
I think we still want to keep the manifest instruction with owner. This was the original reason we had to have the `*WithOwner` invocation in the first place.

Reply:
Happy to provide some insight into this. The two main reasons as to why I made this change are:

1. I find it awkward that `*WithOwner` instructions take a `NonFungibleAddress` for the owner field. It feels like this is remnants of the old model where things could have an owner and that owner was some system-created `NonFungible`. In this new model, the choice to restrict an owner to non-fungibles seems rather strange to me. 
2. With the concept of "owner" now gone, it seems strange that the radix-engine still needs to know about it (in the tx processor)

We could add them back, however, I imagine that a better way would be to say that the owner is some `AccessRule` (not `AccessRules`) instead of being strictly a `NonFungible`.

Reply:
I don't disagree but we should re-visit this with Product as `*WithOwner` with `NonFungibleAddress` was the initial agreement we had.

Reply:
Sounds good, will add this back.